File: test_dir/python/src/if_elif_else.py
def if_elif_else_fn(a: int, b: int) -> str:
    ret: str
    if a < b:
        ret = "less than"
    elif a == b:
        ret = "equal"
    else:
        ret = "greater than"
    return ret


def if_elif_elif_else_fn(a: int, b: int) -> str:
    ret: str
    if a < b:
        ret = "less than"
    elif a == b:
        ret = "equal"
    elif a > b:
        ret = "dunno"
    else:
        ret = "greater than"
    return ret

File: pypp_cli/test_dir/cpp/src/if_elif_else/if_elif_else.cpp
#include "if_elif_else/if_elif_else.h"

namespace me {
pypp::PyStr if_elif_else_fn(int a, int b) {
    pypp::PyStr ret;
    if (a < b) {
        ret = pypp::PyStr("less than");
    } else if (a == b) {
        ret = pypp::PyStr("equal");
    } else {
        ret = pypp::PyStr("greater than");
    }
    return ret;
}

pypp::PyStr if_elif_elif_else_fn(int a, int b) {
    pypp::PyStr ret;
    if (a < b) {
        ret = pypp::PyStr("less than");
    } else if (a == b) {
        ret = pypp::PyStr("equal");
    } else if (a > b) {
        ret = pypp::PyStr("dunno");
    } else {
        ret = pypp::PyStr("greater than");
    }
    return ret;
}

} // namespace me